tests/kernels/test_attention_custom.py

```diff
@@ -286,8 +278,11 @@ def test_paged_attention(
 
     # NOTE(zhaoyang): FP8 KV Cache will introduce quantization error,
     # so we use a relaxed tolerance for the test.
-    atol, rtol = 1e-3, 1e-5
-    atol = 5e-3
+    atol, rtol = 1e-4, 1e-5
+    if dtype == torch.bfloat16: atol, rtol = 2e-4, 1e-5
+    if use_alibi:
+        if dtype == torch.half: atol, rtol = 5e-4, 1e-5
+        if dtype == torch.bfloat16: atol, rtol = 1e-3, 1e-5
```

Comment:
Are we sure about this - bfloat16 needs larger error tolerance?

Reply:
Yes, because bf16 has lower precision than f16. I've kept atol lower than the defaults in test_attention.py wherever possible.